crates/bevy_picking/src/lib.rs

```diff
@@ -0,0 +1,212 @@
+//! TODO, write module doc
+
+#![deny(missing_docs)]
+
+pub mod backend;
+pub mod pointer;
+
+use bevy_app::prelude::*;
+use bevy_ecs::prelude::*;
+use bevy_reflect::prelude::*;
+
+/// Used to globally toggle picking features at runtime.
+#[derive(Clone, Debug, Resource, Reflect)]
+#[reflect(Resource, Default)]
+pub struct PickingPluginsSettings {
+    /// Enables and disables all picking features.
+    pub is_enabled: bool,
+    /// Enables and disables input collection.
+    pub is_input_enabled: bool,
+    /// Enables and disables updating interaction states of entities.
+    pub is_focus_enabled: bool,
+}
```

Comment:
This is a bit ancient, we should revisit if this pattern is still best practice for plugin config.

Reply:
Move into the plugin struct probably?

Reply:
Yup! Cart mentioned in the preferences PR that plugin structs are now accessible at runtime? I'm not really sure how that works though.